pkgs/servers/http/nginx/CVE-2021-23017.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/src/core/ngx_resolver.c b/src/core/ngx_resolver.c
```

Comment:
Can you add the reference to the source of the patch as comment at the top of this file, please?

Reply:
Obsolete.